WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountPercentageView.swift

```diff
@@ -4,32 +4,27 @@ struct AddCustomAmountPercentageView: View {
     @ObservedObject private(set) var viewModel: AddCustomAmountPercentageViewModel
 
     var body: some View {
-        VStack(alignment: .leading, spacing: Layout.mainVerticalSpacing) {
-            HStack() {
-                Text(Localization.percentageInputTitle)
-                    .font(.subheadline)
-                    .foregroundColor(Color(.textSubtle))
+        Group {
+            VStack(alignment: .leading) {
+                LabeledContent {
+                    Text("(\(viewModel.baseAmountForPercentageString))")
```

Comment:
This one is pre-existing:
```suggestion
                    Text(viewModel.baseAmountForPercentageString)
```